### PR TITLE
Add PhantomJS2 to Karma browser list, fix Karma warnings

### PR DESCRIPTION
**Pull request by ryzy**

### CODE REVIEW

**Comment by ryzy** [comment]
See discussion in #137 - not sure if that's the ideal/final solution, but at least it works...

File: test-main.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ System.config({
   defaultJSExtensions: true,
   paths: {
     'angular2/*': 'node_modules/angular2/*.js',
-    '@reactivex/rxjs/*': 'node_modules/@reactivex/rxjs/*.js',
+    '@reactivex/rxjs/*': 'node_modules/angular2/node_modules/@reactivex/rxjs/*.js'
```
Comment (mgechev) [inline test-main.js]: Are you using npm 3.x.x? It is one of the requirements for the project, I think that with the flat module structure of it this change is not required.

**Comment by mgechev** [comment]
@ryzy awesome job! I left a single comment, since we're coupled with the module structure now.

**Comment by ryzy** [comment]
@mgechev : thanks for the comment, yeah, I was using npm 2.x, shame on me ;)

Indeed, no need for mapping @reactivex, I've reverted appropriate lines.
Plus rebase.

Please review again, thanks!

**Comment by mgechev** [comment]
@ryzy looks great! Thanks for the PR!

**Pull request merged**